### DISCUSSION TITLE
【分享】铯图鉴赏

### DISCUSSION THREAD

**Discussion by capacitor1** [share121/inter-knot#1485] [其他]
![image](https://github.com/user-attachments/assets/6acc87e0-0f57-46ff-8d9a-3a954bb574ff)

看！**铯图！**

![image](https://github.com/user-attachments/assets/04d2227d-8c2b-4903-8b76-a11ed2682525)
![image](https://github.com/user-attachments/assets/63a839e1-75fd-4a06-a4fa-78312f49f569)
![image](https://github.com/user-attachments/assets/2c94c158-d445-48c1-ab7e-7ada3a5fb773)
![image](https://github.com/user-attachments/assets/c8bf9aee-ef45-496b-a77e-870ba3b6bb78)
![image](https://github.com/user-attachments/assets/649ad6d3-02d0-43a7-8e35-9e2f6a38c3d9)

百度百科：铯，元素符号Cs，原子序数为55，位于第六周期，IA族，其单质是一种淡金黄色的活泼金属，熔点低，在空气中极易被氧化，能与水剧烈反应生成[氢气](https://baike.baidu.com/item/%E6%B0%A2%E6%B0%94/1760269?fromModule=lemma_inlink)且爆炸 [1]。铯在自然界没有[单质](https://baike.baidu.com/item/%E5%8D%95%E8%B4%A8/1143527?fromModule=lemma_inlink)形态，仅以[盐](https://baike.baidu.com/item/%E7%9B%90/16602623?fromModule=lemma_inlink)的形式极少的分布于陆地和海洋中。铯是制造真空件器、[光电管](https://baike.baidu.com/item/%E5%85%89%E7%94%B5%E7%AE%A1/9005290?fromModule=lemma_inlink)等的重要材料。放射性核素Cs-137是日本福岛第一核电站泄漏出的放射性污染中的一种。
铯是已知元素中（包括[放射性元素](https://baike.baidu.com/item/%E6%94%BE%E5%B0%84%E6%80%A7%E5%85%83%E7%B4%A0/11043501?fromModule=lemma_inlink)）[金属性](https://baike.baidu.com/item/%E9%87%91%E5%B1%9E%E6%80%A7/1775204?fromModule=lemma_inlink)最强的（注意不是[金属活动性](https://baike.baidu.com/item/%E9%87%91%E5%B1%9E%E6%B4%BB%E5%8A%A8%E6%80%A7/1772139?fromModule=lemma_inlink)，活动性最强的是[锂](https://baike.baidu.com/item/%E9%94%82/400413?fromModule=lemma_inlink)）。 [2]

doge

**Comment by BaiLangLAINA** [comment]
太铯力

**Comment by dudu-ke** [comment]
布什·搁们

**Comment by Fairy561** [comment]
逆天

**Comment by otmafo** [comment]
使我的头旋转

**Comment by lllliufeng** [comment]
好好好

**Comment by bin-lin2** [comment]
好铯，爱看，多来点（

**Comment by WarrenCham** [comment]
豪铯，豪砍，铎铼钿

**Comment by DS6AWWDZ9326** [comment]
这个委托我接了！

**Comment by Ste11er** [comment]
好铯

**Comment by FFClub-ZT** [comment]
多来点！

**Comment by Loxia1121** [comment]
好有铯力

**Comment by aizai1173** [comment]
驱动盘都卸了，就给我看这个...

**Comment by adsadk12** [comment]
好铯！！！！！

**Comment by laoyee234** [comment]
非常好铯图，这使我脑袋旋转——爱来自瓷器

**Comment by LYBQ** [comment]
斯！出来惹

**Comment by CloudSwordSage** [comment]
好铯好铯！！

**Comment by kbf473900** [comment]
不什戈门
